scripts/bcm_add_lite_nodes.py

```
@@ -0,0 +1,192 @@
+
+import argparse
+import json
+import ssl
+import sys
+import uuid
+from urllib.request import Request, urlopen
+
+import yaml
+
+
+def make_ssl_context(cert, key, verify=False):
```

Ruff (0.15.14)

[warning] Boolean default positional argument in function definition

(FBT002)